ethclient/ethclient_test.go

```diff
@@ -809,7 +809,8 @@ var allRPCMethods = []string{
 	"personal_unlockAccount",
 	"personal_unpair",
 	"trace_block",
-	"trace_filter",
```

Comment:
Why is `trace_filter` removed?

Reply:
Because it is subscription based.

Reply:
I'm pretty sure there's a way to keep this in the docs/OpenRPC generation. 

`eth_subscribe` establishes subscriptions with a different signature pattern (eg. `eth_subscribe ["newHeads"]` -> `(*api.Eth) SubscribeNewHeads(...) (rpc.Subscription, error)`.
So if this convoluted pattern can do it, then `trace_filter` must be able to too :wink:. 

It might require some noodling in node/openrpc.go.

Reply:
That's great. I will have a look at it. 
Do you think it worths handling this in a next PR, so as we merge this one? As we have to add the remaining bits for `trace_fitler` as well in a next PR.

Reply:
@meowsbits can you please let me know if this is what you had in mind? #345